app/src/main/java/com/alkemy/ongandroid/core/Utils.kt

```diff
@@ -1,9 +1,18 @@
 package com.alkemy.ongandroid.core
 
+import android.app.Activity
 import android.content.Context
 import android.widget.Toast
+import androidx.fragment.app.Fragment
 
-fun Any.toast(context: Context, message:String, isLong: Boolean = false){
+fun Activity.toast(context: Context, message:String, isLong: Boolean = false){
+    if (isLong){
+        Toast.makeText(context, message, Toast.LENGTH_LONG).show()
+    }else{
+        Toast.makeText(context, message, Toast.LENGTH_SHORT).show()
+    }
+}
+fun Fragment.toast(context: Context, message:String, isLong: Boolean = false){
```

Comment:
aca al idea de hacer una extension function de cada uno es no pasar el parametro de `context` y en el caso de la Activity usar `this` y en el caso del Fragment usar `requireContext()`